Kubernetes/IdentityService/test-isapi-kubernetes-manifest.yml

```diff
@@ -97,7 +97,7 @@ spec:
         - name: DATAPROTECTION__APPLICATIONNAME
           value: "ISAPI Test"
         - name: PUBLICBROWSE__BASEADDRESS
-          value: https://buyingcataloguedev.uksouth.cloudapp.azure.com
+          value: https://buyingcataloguetest.uksouth.cloudapp.azure.com
```

Comment:
it'd be public, wouldn't it?
https://buyingcataloguetestpublic.uksouth.cloudapp.azure.com